.github/workflows/newsfragment_checker.yml

```diff
@@ -0,0 +1,36 @@
+name: newsfragment_checker
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
```

Comment:
Is there a reason for using Python 3.7 here? Because the workflow originally used 3.9 and I don't see any reason to downgrade the version

Reply:
I think to be honest there is no reason at all for this workflow to use any particular version. I will let it use the default version.